app/src/main/res/layout/activity_main.xml

```diff
@@ -14,10 +14,15 @@
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
+
     <com.google.android.material.bottomnavigation.BottomNavigationView
         android:id="@+id/bottom_navigation"
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
+        android:layout_width="413dp"
+        android:layout_height="68dp"
+        android:background="@color/white"
+        app:itemRippleColor="#8C9DDF"
```

Comment:
an idea for polishing in later weeks: replace hardcoded colors with themes/styles. themes allow you to define all your colors and whatnot in one place which makes it easy to, for example, create a "light mode" theme and a "dark mode" theme